.github/workflows/go.yml

```diff
@@ -0,0 +1,33 @@
+name: Go
+on:
+  push:
+    tags:
+      - v*
+    branches: [ master ]
```

Comment:
There is no _master_ branch - only _main_

Reply:
indeed, copy and paste error, fixed